scripts/db_migrator.py

```diff
@@ -525,9 +525,24 @@ def version_2_0_0(self):
 
     def version_2_0_1(self):
         """
-        Current latest version. Nothing to do here.
+        Version 2_0_1.
         """
         log.log_info('Handling version_2_0_1')
+        warmreboot_state = self.stateDB.get(self.stateDB.STATE_DB, 'WARM_RESTART_ENABLE_TABLE|system', 'enable')
+
+        if warmreboot_state != 'true':
```

Comment:
what is the behavior of lacp key for warm reboot?

Reply:
In case that the upgrade was done using warm-reboot we are not updating the DB.
This is done because changing the lacp key of the existing port channels will cause the link to go down which we can't do during warm reboot.